native/packages/react-native-bpk-component-image/src/BpkImage.js

```diff
@@ -0,0 +1,176 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Animated,
+  StyleSheet,
+  View,
+  ViewPropTypes,
+  type StyleObj,
+} from 'react-native';
+import BpkSpinner from 'react-native-bpk-component-spinner';
+
+import {
+  animationDurationBase,
+  borderRadiusSm,
+  colorGray300,
+} from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  outer: {
+    width: '100%',
+    height: '100%',
+  },
+  outerWithBorderRadius: {
+    borderRadius: borderRadiusSm,
+    overflow: 'hidden',
+  },
+  image: {
+    width: '100%',
+    height: '100%',
+    position: 'absolute'
+  },
+  loadingIndicatorView: {
+    width: '100%',
+    height: '100%',
+    backgroundColor: colorGray300,
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'absolute',
```

Comment:
I'm not certain but I reckon the number of style props can be reduced here - is `position: absolute` needed? I'll have a play locally and see if I can optimise it, if you don't mind?

Reply:
Just had a play with this locally and this achieves the same thing in fewer lines of code:

```
  loadingIndicatorView: {
    flex: 1,
    backgroundColor: colorGray300,
    justifyContent: 'center',
    alignItems: 'center',
  },
```